src/Http/Cookie/Cookie.php

```diff
@@ -416,15 +417,15 @@ protected function validateName(string $name): void
     /**
      * @inheritDoc
      */
-    public function getValue()
+    public function getValue(): array|string
     {
         return $this->value;
     }
 
     /**
      * @inheritDoc
      */
-    public function getScalarValue()
+    public function getScalarValue(): array|string
```

Comment:
This one doesn't make much sense. The value is still `array|string` not `mixed` and certainly isn't converting anything.

Reply:
I was returning mixed. I'm not sure if static analyzer just ignore undefined return types though.

Reply:
The method calls `_flatten()` if the value is expanded (aka an array) which has return type `string`, so the return type here should should be a `string`.